libmscore/tempotext.cpp

```diff
@@ -31,6 +31,7 @@ TempoText::TempoText(Score* s)
       {
       _tempo      = 2.0;      // propertyDefault(P_TEMPO).toDouble();
       _followText = false;
+      _relative = 0;
```

Comment:
1.0 would be better indeed. If we need to compare float in test cases, we can http://doc.qt.io/qt-5/qtglobal.html#qFuzzyCompare-1 but it's better to avoid = comparison of float in the main code.